pandas/tests/io/json/test_normalize.py

```diff
@@ -287,6 +287,32 @@ def test_shallow_nested(self):
         expected = DataFrame(ex_data, columns=result.columns)
         tm.assert_frame_equal(result, expected)
 
+    def test_nested_meta_path_with_nested_record_path(self, state_data):
+        # GH 27220
+        result = json_normalize(
+            state_data,
+            ["counties", "name"],
+            ["state", "shortname", ["info", "governor"]],
+            errors="ignore",
+        )
+        ex_data = {
+            0: [
+                i
+                for word in ["Dade", "Broward", "Palm Beach", "Summit", "Cuyahoga"]
+                for i in word
+            ],
+            "state": ["Florida"] * 21 + ["Ohio"] * 14,
+            "shortname": ["FL"] * 21 + ["OH"] * 14,
+            "info.governor": ["Rick Scott"] * 21 + ["John Kasich"] * 14,
+            "population": [12345] * 4
+            + [40000] * 7
+            + [60000] * 10
+            + [1234] * 6
+            + [1337] * 8,
+        }
+        expected = DataFrame(ex_data, columns=result.columns)
```

Comment:
can the columns be specified explicitly and not dependent on the result?

Reply:
Good suggestion!